LLama/LLamaExecutorBase.cs

```diff
@@ -315,6 +315,34 @@ public virtual async IAsyncEnumerable<string> InferAsync(string text, IInference
             }
         }
 
+        /// <summary>
+        /// Asynchronously runs a prompt through the model to compute KV cache without generating any new tokens.
```

Comment:
```suggestion
        /// Asynchronously runs a prompt through the model to compute KV cache without generating any new tokens.
        /// It could reduce the latent of the first time response if the first input from the user is not immediate.
```